src/main/java/org/jitterbit/connector/dropbox/activities/ProcessFileActivity.java

```diff
@@ -216,6 +281,29 @@ private void init() {
     model.schemaContentType = org.jitterbit.connector.sdk.metadata.SchemaMetaData.SchemaContentType.XML;
     model.filename = obj.getObjectName() + ".xml";
 
+    model = new DiscoverableObjectModel();
```

Comment:
I think you need a

`objects.add(model);` 

before line 284 as you are redefining what `model` is and discarding the previous definition.